package/scripts/common/quarto

```diff
@@ -71,26 +67,28 @@ else
   # Turn of type checking for bundled version
   QUARTO_DENO_OPTIONS=--no-check
 
-  # If Quarto is bundled into an `.app` file, it will be looking for the 
+  # If Quarto is bundled into an `.app` file, it will be looking for the
   # share directory over in the resources folder.
-  if [[ $SCRIPT_PATH == *"/Contents/MacOS/quarto/bin" ]]; then
-    export QUARTO_SHARE_PATH="`cd "$SCRIPT_PATH/../../../Resources/quarto/share";pwd`"
-  elif [[ $SCRIPT_PATH == *"/usr/local/bin/quarto" ]]; then
-    export QUARTO_SHARE_PATH="`cd "$SCRIPT_PATH/../../share/quarto";pwd`"
-  else
-    export QUARTO_SHARE_PATH="`cd "$SCRIPT_PATH/../share";pwd`"
+  if [ -z "${QUARTO_SHARE_DIR+x}" ]; then
+    if [[ $SCRIPT_PATH == *"/Contents/MacOS/quarto/bin" ]]; then
+      export QUARTO_SHARE_DIR="`cd "$SCRIPT_PATH/../../../Resources/quarto/share";pwd`"
+    elif [[ $SCRIPT_PATH == *"/usr/local/bin/quarto" ]]; then
+      export QUARTO_SHARE_DIR="`cd "$SCRIPT_PATH/../../share/quarto";pwd`"
+    else
+      export QUARTO_SHARE_DIR="`cd "$SCRIPT_PATH/../share";pwd`"
+    fi
   fi
 
   if [ "$1" == "--version" ] || [ "$1" == "-v" ]; then
-    echo `cat "$QUARTO_SHARE_PATH/version"`
+    echo `cat "$QUARTO_SHARE_DIR/quarto_version"`
```

Comment:
Sadly the RStudio IDE also relies on the presence of the `version` file so we can't readily rename this.

Reply:
We'll need another place then, perhaps with a different env var. We can't just drop a file like that into a shared folder where it's likely to have conflicts. Or maybe I'm misinterpreting the share dir and in the conda package it should be something like `QUARTO_SHARE_DIR=$PREFIX/share/quarto`, and this `version` file would live in there.

Reply:
Yes exactly, the share dir is a quarto specific share dir NOT the global one